packages/legend-extension-external-store-service/src/components/ESService_ServiceStoreEmbeddedData.tsx

```diff
@@ -912,44 +912,45 @@ export const ServiceStoreEmbeddedDataEditor = observer(
                 </button>
               </div>
             </div>
-            <ContextMenu
-              className="service-store-embedded-data-editor__content"
-              disabled={isReadOnly}
-              content={
-                !isReadOnly &&
-                currentServiceStubMapping !== undefined && (
-                  <div
-                    className="service-store-embedded-data-editor__context-menu"
-                    onClick={deleteServiceStubMapping(
-                      currentServiceStubMapping,
-                    )}
-                  >
-                    Delete
-                  </div>
-                )
-              }
-              menuProps={{ elevation: 7 }}
-            >
-              <div className="service-store-embedded-data-editor__content__lists">
-                <MenuContent className="service-store-embedded-data-editor__dropdown">
-                  {serviceStoreEmbeddedDataState.embeddedData.serviceStubMappings.map(
-                    (serviceStubMapping: ServiceStubMapping, index: number) => (
+
+            <div className="service-store-embedded-data-editor__content__lists">
+              <MenuContent className="service-store-embedded-data-editor__dropdown">
```

Comment:
I believe this is not the idiomatic way of using `Menu` component. Usually, you use `Menu` in the `content` of `ContextMenu`. 

e.g.

```tsx
<ContextMenu
      className="virtual-assistant__contextual-support"
      disabled={!contextualEntry}
      menuProps={{
        elevation: 7,
        classes: {
          root: 'virtual-assistant__context-menu',
        },
      }}
      content={
        <MenuContent>
          <MenuContentItem onClick={copyContextIDToClipboard}>
            Copy Context ID
          </MenuContentItem>
          <MenuContentItem onClick={copyCurrentContextIDToClipboard}>
            Copy Current Context ID
          </MenuContentItem>
          <MenuContentItem onClick={copyContextStackToClipboard}>
            Copy Context Stack
          </MenuContentItem>
        </MenuContent>
      }
    >
      {...}
</ContextMenu>
```

See `VirtualAssistant` for example.

This causes the styling to be weird. If you follow the idiomatic usage, you might not need styling at all. Let's see.